### PR TITLE
PLANET-6963 Enable navigation dropdown submenus by default

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [PLANET-6963](https://jira.greenpeace.org/browse/PLANET-6963)
For this we need to remove the corresponding feature flag and the obsolete code linked to it

### Testing

The feature flag should be removed from the `Planet 4 > Information Architecture` settings, and the navigation bar should look as expected with or without dropdown menu items.

### CODE REVIEW

File: templates/navigation-bar.twig [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {% set header_classes = 'top-navigation navbar'
 	~ (custom_styles.nav_border == 'border' ? ' navigation-bar_border' : '')
-	~ (mobile_tabs_menu ? ' with-mobile-tabs' : '')
-	~ (dropdown_menu ? ' with-dropdown-menu' : '') %}
```
Comment (mleray) [inline templates/navigation-bar.twig:4]: This class was not used

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [pandora](https://www-dev.greenpeace.org/test-pandora) | [admin](https://www-dev.greenpeace.org/test-pandora/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-pandora/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-pandora) | [composer-local.json](https://github.com/greenpeace/planet4-test-pandora/blob/main/composer-local.json)

:watch: 2023.01.20 11:47:54

**Review by lithrel — APPROVED** [review]
:+1: 
You could also add a patch to remove this option from databases (like [this one](https://github.com/greenpeace/planet4-master-theme/blob/main/src/Migrations/M012RemoveThemeEditorOption.php), with proper option key and name), or we can create a ticket for later

File: src/Settings/Features.php [hunk 2]
```diff
@@ -9,7 +9,6 @@
 use P4\MasterTheme\Features\Dev\BetaBlocks;
 use P4\MasterTheme\Features\Dev\CoreBlockPatterns;
 use P4\MasterTheme\Features\Dev\DisableDataSync;
-use P4\MasterTheme\Features\Dev\DisableTagRedirectPages;
```
Comment (mleray) [inline src/Settings/Features.php:12]: This was unused

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**